verification/verifier/verifier.go

```diff
@@ -78,3 +78,11 @@ func (o *Verifier) ProcessEvidence(
 
 	return appraisalCtx.Result, nil
 }
+
+func (o *Verifier) GetPublicKeys() (*proto.PublicKey, error) {
+	k, err := o.VTSClient.GetEARSigningPublicKey(context.Background(), &emptypb.Empty{})
+	if err != nil {
+		return nil, err
+	}
+	return k, err
```

Comment:
Could be simplified to:

```suggestion
	return o.VTSClient.GetEARSigningPublicKey(context.Background(), &emptypb.Empty{})
```